instrumentation/logback/logback-appender-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/appender/v1_0/LoggingEventToReplay.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.logback.appender.v1_0;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.classic.spi.IThrowableProxy;
+import ch.qos.logback.classic.spi.LoggerContextVO;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Marker;
+import org.slf4j.event.KeyValuePair;
+
+class LoggingEventToReplay implements ILoggingEvent {
+
+  private final ILoggingEvent loggingEvent;
+  private final long timeStamp;
+  private StackTraceElement[] callerData;
+  private String threadName;
```

Comment:
can you add a comment here why we need to capture these 3 attributes (instead of delegating to `loggingEvent`)?